config/resources/configs/frequency-testnet-paseo.json

```diff
@@ -0,0 +1,41 @@
+{
+  "sdkMaxUsersGraphSize": 10000,
+  "sdkMaxStaleFriendshipDays": 90,
+  "maxGraphPageSizeBytes": 1024,
+  "maxPageId": 16,
+  "maxKeyPageSizeBytes": 65536,
+  "graphPublicKeySchemaId": 7,
+  "schemaMap": [
+    [
+      8,
+      [
+        "1.0",
+        {
+          "connectionType": "follow",
+          "privacyType": "public"
+        }
+      ]
+    ],
+    [
+      9,
+      [
+        "1.0",
+        {
+          "connectionType": "follow",
+          "privacyType": "private"
+        }
+      ]
+    ],
+    [
+      10,
+      [
+        "1.0",
+        {
+          "connectionType": "friendship",
+          "privacyType": "private"
+        }
+      ]
+    ]
+  ],
```

Comment:
On the paseo schema 10 does not have any settings , expecting signature required setting on it ?

Reply:
Same with schema 9 it has setting set to 0 , I think private schemas require signatures enforcement

Reply:
schema 8 is good it's Avro Paginated and setting=0

Reply:
I think schema 7 has setting = 3 not sure if it corresponds to `AppendOnly`

Reply:
These are the same settings as appear on mainnet. So if they are wrong, so is mainnet 😟

Reply:
Yeah I think they are wrong "in spirit" meaning most stateful operations everywhere is only calling "upsert_page" extrinsic and it does not check for schema permission but extrinsics with "...._with_signature" will fail 

And I think delete page extrinsic will fail